src/components/details-card/Link.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+
+export const ExternalLink: React.FC<ExternalLinkProps> = ({
+    children,
+    href,
+    text,
+    dataTestID,
+    stopPropagation,
+  }) => (
+    <a
+      className='co-external-link'
```

Comment:
I dont see this class defined anywhere.

Reply:
Will it use the console common style in run-time? I see it defined in console _common.scss

Reply:
Yes if its defined there, then it should be fine.